opendevin/controller/state/state.py

```diff
@@ -98,6 +98,8 @@ class State:
     traffic_control_state: TrafficControlState = TrafficControlState.NORMAL
     # global metrics for the current task
     metrics: Metrics = Metrics()
+    # local metrics for the current subtask
+    local_metrics: Metrics = Metrics()
```

Comment:
I find `local_` confusing here. As you write in the comment, `subtask_` would be more fitting?

Reply:
unfortunately `subtask_` is also confusing because we haven't fully removed support for PlannerAgent yet, which has explicit concepts of tasks and subtasks and are different from what I wrote in the comment

Reply:
Ok, understood.

Reply:
But we probably should do the renaming once we get rid of PlannerAgent